modules/rte_mis_student_tracking/rte_mis_student_tracking.module

```diff
@@ -4,3 +4,45 @@
  * @file
  * Contains rte_mis_allocation.module.
  */
+
+/**
+ * Callback to get the list of allowed values.
+ */
+function rte_mis_student_tracking_get_allowed_values($definition, $entity = NULL) {
```

Comment:
Funcation param are missing

Reply:
Also why we are creating multiple functions for academic years

Reply:
We wanted to keep the code for a particular module in that module itself. As it will be easier for the future developers to understand the code base.

Reply:
I have added the function params & proper doc comment

Reply:
@RohanSingh-inno The code is same in both the modules, right?

Reply:
If that is the case then can we have a common function in the .inc file?

Reply:
@kunal-sin fields are not same, fields like grade, current class, entry year are not present in '.module' file of the allocation module. Both the modules have only one field in common 'academic session'. Apart from that fields are different.